cypress/component/ColumnOrderIcons.cy.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react"
+import ColumnOrderIcons from "../../src/features/CourtCaseFilters/ColumnOrderIcons"
+
+describe("ColumnOrderIcons.cy.tsx", () => {
+  it("up arrow", () => {
+    cy.mount(<ColumnOrderIcons columnName={"defendantName"} currentOrder={"asc"} orderBy={"defendantName"} />)
```

Comment:
could we add some assertions here that an element with the class `upArrow` exists? and likewise for the other tests in this file, then we're checking that the right arrows are showing